bedrock/privacy/templates/privacy/notices/firefox-2025.html

```diff
@@ -74,3 +74,8 @@ <h3>{{ ftl('privacy-firefox-contents') }}</h3>
 
 {# Exclude stub attribution for in-product pages: issus 9620 #}
 {% block stub_attribution %}{% endblock %}
+
+{% block js %}
```

Comment:
should be running in head to avoid all the other stuff loading before (i.e. css, html)

Reply:
I moved it up to `extrahead` block. I think these are sufficient enough edge cases that it's not worth going higher and interfering with the people who are here to access this content.